.github/workflows/release-alpha.yml

```diff
@@ -84,12 +84,12 @@ jobs:
           push: true
           context: .
           file: ./deployments/Dockerfile
-          tags: horuszup/horusec-cli:alpha
+          tags: wiliansilvazup/horusec-cli:alpha
```

Comment:
I believe you forgot to change the docker hub repository.

Reply:
oh yeah tnks